third_party/silabs/efr32_sdk.gni

```diff
@@ -46,10 +46,16 @@ declare_args() {
   # Enable Segger System View
   use_system_view = false
 
-  # ICD Configuration flags
-  sl_ot_idle_interval_ms = 30000  # 30s Idle Intervals
-  sl_ot_active_interval_ms = 200  # 500ms Active Intervals
-  sl_active_mode_threshold = 1000  # 1s Active mode threshold
+  # ICD Openthread Configuration flags
```

Comment:
It looks like sl_ot_idle_interval_ms sl_ot_idle_interval_ms aren't part of the ICD cluster data model and spec. Are these values described elsewhere?